CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ pkg_check_modules(DEPS REQUIRED
   gtksourceview-3.0>=3.10
   gthread-2.0
   gtk+-3.0>=3.9.10
-  webkit2gtk-3.0
+  PKG_CHECK_MODULES([webkit2gtk-3.0], [webkit2gtk-4.0])
```

Comment:
On Archlinux, `webkit2gtk-3.0` is not available but `webkit2gtk-4.0` is available. So I added that.